controller/task_queue/handler/embedding.py

```diff
@@ -34,15 +37,30 @@ def __start_task(task: Dict[str, Any]) -> bool:
 
     user_id = task["created_by"]
     attribute_id = task["task_info"]["attribute_id"]
-    embedding_handle = task["task_info"]["embedding_handle"]
-    if task["task_info"]["embedding_type"] == EmbeddingType.ON_ATTRIBUTE.value:
-        embedding_manager.create_attribute_level_embedding(
-            project_id, user_id, attribute_id, embedding_handle
-        )
-    else:
-        embedding_manager.create_token_level_embedding(
-            project_id, user_id, attribute_id, embedding_handle
-        )
+    embedding_type = task["task_info"]["embedding_type"]
+    embedding_name = task["task_info"]["embedding_name"]
+    platform = task["task_info"]["platform"]
+    model = task["task_info"]["model"]
+    api_token = task["task_info"]["api_token"]
```

Comment:
since we don'T always have an api_token/model I was expecting some condition logic or a dictionary get

Reply:
sorry misunderstanding the get is done in the mutation -> can be resovled